### ISSUE TITLE
Capitalize first letter of all languages (dropdown)

### ISSUE THREAD

**Issue by RIMOPA** [ScratchAddons/website-v2#163]
Now, some languages have the first letter capitalized (English, Nederlands, Türkçe and Tiếng Việt) and some don't have it (español, français, italiano, polski, português (Brasil), русский and slovenščina). it's ugly that some have it and some don't, so I propose capitalize the first letter of all languages.

Now:
![image](https://user-images.githubusercontent.com/97112832/155893848-00f7ba66-670c-4638-98d0-52a8ef33c799.png)

Idea (First letter capitalized):
![image](https://user-images.githubusercontent.com/97112832/155893731-b89d3290-cfa8-4c6f-952f-d6eaa60caf7e.png)

**Comment by Secret-chest** [comment]
Because the language name is text in that language, and in some languages language names are not proper nouns. For example, to say 'Romanian language" in Romanian, you would say "limba română". That is correct.

**Comment by RIMOPA** [comment]
> Because the language name is text in that language, and in some languages language names are not proper nouns. For example, to say 'Romanian language" in Romanian, you would say "limba română". That is correct.

Even if they're not proper nouns, I still think it's very ugly; worth it?

**Comment by WorldLanguages** [comment]
I agree, it looks ugly, and Scratch capitalizes the names of the languages.

**Comment by mxmou** [comment]
> > I agree, it looks ugly, and Scratch capitalizes the names of the languages.
> 
> But, in some languages language names are not proper nouns.

That doesn't mean capitalizing them is wrong, it just isn't necessary.

**Comment by WorldLanguages** [comment]
> But, in some languages language names are not proper nouns.

Including my own native language, español. However, seeing it capitalized in a dropdown never looked wrong to me.

**Comment by Hans5958** [comment]
For the language names, I'm using Intl.DisplayNames, so it is a standard. English and other languages are capitalized, but some aren't. This is intentional, for cultural reasons I presume.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DisplayNames

Currently, I'm looking at my choices for the sake of accuracy. I think I'm going for a library at this moment.

**Comment by RedGuy12** [comment]
> For the language names, I'm using Intl.DisplayNames, so it is a standard. English and other languages are capitalized, but some aren't. This is intentional, for cultural reasons I presume.

wait a sec...that's only supported in chrome 81+ but sa theoretically supports chrome 80+...

**Comment by Hans5958** [comment]
> > For the language names, I'm using Intl.DisplayNames, so it is a standard. English and other languages are capitalized, but some aren't. This is intentional, for cultural reasons I presume.
> 
> wait a sec...that's only supported in chrome 81+ but sa theoretically supports chrome 80+...

It's added server side (aka on CI build) so it uses Node.js 16.
